runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java

```diff
@@ -145,6 +148,7 @@
 
 	static
 	{
+		PATTERN = Pattern.compile("^[A-Za-z0-9_\\s]*$");
```

Comment:
Just initialize this while declaring

Reply:
Not sure why I didn't do that lol, probably because it was 4am